app/src/main/res/layout/fragment_new_item.xml

```diff
@@ -36,18 +38,27 @@
             android:textSize="30sp"
             android:textStyle="bold" />
 
-        <ImageView
-            android:id="@+id/new_item_image"
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="150dp"
-            android:contentDescription="@string/item_image_description"
-            android:src="@drawable/ic_baseline_image_24" />
-
-        <Button
-            android:id="@+id/new_item_take_picture"
-            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="@string/take_picture" />
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/new_item_image"
+                android:layout_width="match_parent"
+                android:layout_height="70dp"
+                android:layout_weight="1"
+                android:contentDescription="@string/item_image_description"
+                android:src="@drawable/ic_baseline_image_24" />
+
+            <ImageView
+                android:id="@+id/new_item_take_picture"
+                android:layout_width="match_parent"
+                android:layout_height="70dp"
+                android:layout_weight="1"
+                android:contentDescription="@string/item_image_description"
+                android:src="@android:drawable/ic_menu_camera" />
```

Comment:
Shouldn't we use a ImageButton with an ImageView here, since the image is not supposed to change?
Also I think having it on the right and the same size makes the image we add way too small, so maybe you should put it under it like before.

Reply:
Indeed, I just copied what you already had for the other button.
As for the size, the previous version took too much space (since I'm adding a field the form has grown in size), so I needed to reduce the size of that. Improvements can be made, but this was the quickest fix.

Reply:
Like having one button that opens two other buttons, to permits choosing between using the camera or using a picture from the gallery; instead of showing two buttons

Reply:
I reverted it.

Reply:
I saw your changes, but now the create item button doesn't fit on my emulator (probably too small), can't we just make the whole fragment scrollable so we can always access everything?

Reply:
I reduced the size of the form

Reply:
Alright, but at some point we'll have to make our fragments scrollable so it can be used on all screens. Maybe we can make some user stories for that.

Reply:
Agreed!